compliance_checker/cfunits.py

```diff
@@ -0,0 +1,90 @@
+from pyudunits2 import UnitSystem, UnresolvableUnitException
+
+try:
+    import cf_units
+except ImportError:
+    cf_units = False
+
+
+class PyUdunits2:
+    """Workaround for the differences in pyudunits2 and cf-units.
+
+    NB: Some of these may change and/or get implemented upstream. Pyudunits2 is new and in-flux.
+
+    1/4 Raise the same ValueError to match cf-unit errors.
+    2/4 Creates an empty unit from None to mimic cf-unit's Unit('unknown')
```

Comment:
Would like to explore this one further - it sounds like a bad idea to me (and looks like it isn't a udunits2 concept).  Would you mind opening an issue in pyudunits2 with some thoughts/detail on "unknown" unit usage?

Reply:
> Would you mind opening an issue in pyudunits2 with some thoughts/detail on "unknown" unit usage?

The only reason I wrote that here is b/c we relied on this behavior before. I'm OK dropping it. I agree that it is a bad idea and it is not hard to adapt code that used this to avoid it.